### PR TITLE
Feat(docs): add docs for unpause process with multisig

### LINKED ISSUE #1486: docs: add unpause process with multisig
# Description 

Based on #1482, add the same process for unpause the contracts

### PR DESCRIPTION
**Pull request by PatStiles**
# Add docs for unpause process with multisig

## Description

Closes #1486 

- Add docs for unpause process with multisig
- Fix an environment variable sourcing issue in `deploy_batcher_payment_service`

## Type of change

Please delete options that are not relevant.

- [x] New feature

### CODE REVIEW

**Review by MarcosNicolau — APPROVED** [review]

File: contracts/scripts/deploy_batcher_payment_service.sh
Comment (uri-99) [inline contracts/scripts/deploy_batcher_payment_service.sh:1]: not for this PR

**Review by uri-99 — APPROVED** [review]

File: docs/0_internal/4_b_3_propose_unpause.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+1. Go to [Safe](https://app.safe.global/home)
+
+2. Click on `New transaction` -> `Transaction Builder`
+
+   ![New transaction](./images/4_b_1_unpause_1.png)
+
+   ![Transaction Builder](./images/4_b_1_unpause_2.png)
+
+3. . Get the `AlignedLayerServiceManager` address from ```contracts/script/output/mainnet/alignedlayer_deployment_output.json``` or ```contracts/script/output/holesky/alignedlayer_deployment_output.json``` or ```contracts/script/output/sepolia/alignedlayer_deployment_output.json```
```
Comment (avilagaston9) [inline docs/0_internal/4_b_3_propose_unpause.md]:
```suggestion
3. Get the `AlignedLayerServiceManager` address from ```contracts/script/output/mainnet/alignedlayer_deployment_output.json``` or ```contracts/script/output/holesky/alignedlayer_deployment_output.json``` or ```contracts/script/output/sepolia/alignedlayer_deployment_output.json```
```

File: docs/0_internal/4_b_3_propose_unpause.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+7. Then click on `+ Add new transaction`
+
+   You should see the new transaction to be executed. Then click on `Create batch` to create the transaction.
+
+   ![Add new transaction](./images/4_b_1_unpause_9.png)
+
+8. Review and confirm you are interacting with the correct `BatcherPaymentService` contract and you are calling the `unpause` function.
+
+   ![Review transaction](./images/4_b_1_unpause_10.png)
```
Comment (avilagaston9) [inline docs/0_internal/4_b_3_propose_unpause.md:90]: Not working

File: docs/0_internal/4_b_4_approve_unpause.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+## Approve the UnPause for AlignedLayerServiceManager
+
+1. Go to [Safe](https://app.safe.global/home) and connect your wallet.
+
+2. Go to the `Transactions` tab and find the transaction that was proposed.
+
+3. Get the ```unpause(uint256)``` signature by running:
+
+   ```bash
+   cast calldata "unpause(uint256) <PAUSE_STATE>"
```
Comment (avilagaston9) [inline docs/0_internal/4_b_4_approve_unpause.md]:
```suggestion
   cast calldata "unpause(uint256)" <PAUSE_STATE>
```

**Review by avilagaston9 — APPROVED** [review]
Left some nit comments. Otherwise, LGTM

File: docs/0_internal/4_b_3_propose_unpause.md (hunk 1 above)
Comment (PatStiles) [reply docs/0_internal/4_b_3_propose_unpause.md]: addressed

File: docs/0_internal/4_b_3_propose_unpause.md (hunk 2 above)
Comment (PatStiles) [reply docs/0_internal/4_b_3_propose_unpause.md:90]: addressed

File: docs/0_internal/4_b_4_approve_unpause.md (hunk 3 above)
Comment (PatStiles) [reply docs/0_internal/4_b_4_approve_unpause.md]: addressed

**Review by JuArce — APPROVED** [review]

**Pull request merged**